client/src/tokens/typography.js

```diff
@@ -124,52 +124,62 @@ const typeScale = {
     fontWeight: 'fontWeight.extrabold',
     color: 'colors.primary.DEFAULT',
     lineHeight: 'lineHeight.tight',
+    text: 'Heading level 1 (h1)',
```

Comment:
I recommend we either move this 'text' to the actual storybook file, or rename this to `example` or `description` here.

But I think we are better off being more explicit in the storybook and manually building up the set of example contents.

This might seem more verbose but it is easier to maintain and clearer what is happening in the storybook.